### PR TITLE
removed HTML tags before saving session's info to calendar

### LINKED ISSUE #2096: HTML tags are added to description of calendar on saving any session
**Actual Behaviour**
HTML tags (like `<span>, <p>`) are included in the description of Calendar event on saving any session.
image

**Expected Behaviour**

HTML tags should not be a part of the calendar event description.

**Steps to reproduce it**

Process the string before adding it in intent data

**Would you like to work on the issue?**
YES

### PR DESCRIPTION
**Pull request by simarsingh24**
Fixes #2096

### CODE REVIEW

File: android/app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java [hunk 1]
```diff
@@ -379,7 +379,7 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 Intent intent = new Intent(Intent.ACTION_INSERT);
                 intent.setType("vnd.android.cursor.item/event");
                 intent.putExtra(CalendarContract.Events.TITLE, title);
-                intent.putExtra(CalendarContract.Events.DESCRIPTION, session.getShortAbstract());
+                intent.putExtra(CalendarContract.Events.DESCRIPTION,android.text.Html.fromHtml(session.getShortAbstract()).toString() );
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java]: Please add proper import statement and a space after comma

**Review by iamareebjamal — CHANGES_REQUESTED** [review]

**Review by iamareebjamal — DISMISSED** [review]

**Comment by simarsingh24** [comment]
@iamareebjamal @the-dagger I was solving another issue and accidentally pushed on the same branch. please review

**Comment by iamareebjamal** [comment]
Looks good, can you please squash commits?

**Comment by simarsingh24** [comment]
@iamareebjamal Yeah, sure.

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**